docs/extensions/extension_api.md

```diff
@@ -19,13 +21,13 @@ utility functions for several commonly used structures like collections, http,
 and async. These modules can be imported using the URLs documented in Deno's
 standard library documentation or by downloading them and including individual
 modules as files. All standard library functionality is documented in [Deno's
-standard library docs]
+standard library docs].
 
 [Deno's API docs]: https://doc.deno.land/deno/stable
 [Deno's standard library docs]: https://deno.land/std
 
 ## Phylum API
 
-The Phylum extension API is documented in the [TypeScript module file].
+The Phylum extension API is documented in the [TypeScript module file][api].
 
-[TypeScript module file]: ./cli/src/extension_api.ts
+[api]: https://github.com/phylum-dev/cli/blob/main/cli/src/extension_api.ts
```

Comment:
If you're going to assign the URL a specific key, I'd recommend using one that's going to be unique. Having it called "api" in a file full of documentation about different APIs is going to be confusing and potentially cause issues in the future.

Reply:
Roger that...it will be changed. The choice here was due to line length considerations. It appears that the file was using 80 characters for the width. Numbers are also possible...kind of like a footnotes style. Would that be preferable? Or, would a more descriptive key that causes the line to exceed 80 be acceptable. FWIW...the readme.com rendering allows for lines up to 118 characters before wrapping. GitHub doesn't care about the length and will softwrap anything until it sees 2+ newlines in a row.

Reply:
First of all I don't separated URL links should take line length into consideration. That's kinda the entire point of putting them outside of the block of text. If the URL is longer than 80 columns, you wouldn't be able to use it at all otherwise.

I don't think footnotes like `[1]` would be superior, that would just make things more difficult to follow along. This also would likely only work if you put all the links at the bottom of the entire page. I personally think it's nicer to have them after each paragraph with more descriptive IDs.

If you still want to give it an ID and it **must** absolutely be below 80 characters total, I'd go with `ext_api` at least. But I think we shouldn't care about the column limitation.

Reply:
Agreed...column limitation shouldn't matter. The thought was not that the output would exceed 80 characters in the rendered form, but that the view in the IDE would...and didn't conform to the existing (although unspoken) limit of 80. It will be put back to the original form:

```
[TypeScript module file]: https://github.com/phylum-dev/cli/blob/main/cli/src/extension_api.ts
```

Reply:
> The thought was not that the output would exceed 80 characters in the rendered form, but that the view in the IDE would...and didn't conform to the existing (although unspoken) limit of 80.

Honestly I don't find this relevant at all. IDEs should soft-wrap if the line is too long and even if it was completely cut off it's just a random URL with the important description of what it links to being right at the start; clearly visible.

So ultimately even if it's a thousand columns wide, it doesn't matter, because nobody actually is expected to read the URL. But you're expected to read the paragraphs.